tools/plugins/com.liferay.ide.server.tomcat.core/src/com/liferay/ide/server/tomcat/core/PortalTomcatBundle.java

```diff
@@ -77,13 +79,39 @@ public IPath getAppServerLibGlobalDir() {
 
 	@Override
 	public IPath getAppServerPortalDir() {
-		IPath retval = null;
+		IPath appBasePath = bundlePath.append("webapps");
```

Comment:
All of these checks should also be done during the 'init', so the first line in this method should be: ```
if ((_appServerPortalDir == null) || FileUtil.notExists(_appServerPortalDir.toFile())) {
		```

Can you update it?  Thanks.

Reply:
OK, will send a new pull request